AC3.2Unit2FinalAssessment/AC3.2Unit2FinalAssessment/OtherViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  OtherViewController.swift
+//  AC3.2Unit2FinalAssessment
+//
+//  Created by Tong Lin on 10/6/16.
+//  Copyright © 2016 Tong Lin. All rights reserved.
+//
+
+import UIKit
+
+class OtherViewController: UIViewController, UITextFieldDelegate{
+
+    @IBOutlet weak var slider: UISlider!
+    @IBOutlet weak var stepper: UIStepper!
+    @IBOutlet weak var label: UILabel!
+    @IBOutlet weak var textField: UITextField!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+        _ = self.setting()
+        
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    // MARK: - Navigation
+
+    @IBAction func sliderUpdate(_ sender: UISlider) {
+        self.stepper.value = Double(sender.value)
+        self.label.text = String(sender.value)
+        self.textField.text = String(sender.value)
+    }
+    
+    @IBAction func stepperUpdate(_ sender: UIStepper) {
+        self.slider.value = Float(sender.value)
+        self.label.text = String(sender.value)
+        self.textField.text = String(sender.value)
+    }
+    
+    func textFieldDidEndEditing(_ textField: UITextField) {
+        print("did end")
+    }
+    
+    internal func textFieldUpdate() -> Bool{
```

Comment:
Why is this function returning a Bool?

This function that updates all textfields and labels is a good idea but it's not effective if you don't give it parameters to pass data. The below would work better:
`
 internal func textFieldUpdate(_ value:Int) {
        self.slider.value = Float(value)
        self.stepper.value = Double(value)
        self.label.text = String(value)
    }
`